README.md

```diff
@@ -1,21 +1,51 @@
+# `cleanlab` Examples
 
-Not sure where to start? Try checking out how to find [ImageNet Label Errors](imagenet/imagenet_train_label_errors.ipynb).
+This repo contains code examples that demonstrate how to use [cleanlab](https://github.com/cleanlab) and how [confident learning](https://arxiv.org/abs/1911.00068) works to find label errors.
 
+## Latest Examples
 
-A brief description of the files and folders:
-* `imagenet`, 'cifar10', 'mnist' - code to find label errors in these datasets and reproduce the results in the [confident learning paper](https://arxiv.org/abs/1911.00068). You will also need to `git clone` [confidentlearning-reproduce](https://github.com/cgnorthcutt/confidentlearning-reproduce).
-  - [imagenet_train_crossval.py](imagenet/imagenet_train_crossval.py) - a powerful script to train cross-validated predictions on ImageNet, combine cv folds, train with on masked input (train without label errors), etc.
-  - [cifar10_train_crossval.py](cifar10/cifar10_train_crossval.py) - same as above, but for CIFAR.
-* `classifier_comparison.ipynb` - tutorial showing `cleanlab` performance across 10 classifiers and 4 dataset distributions.
-* `iris_simple_example.ipynb` - tutorial showing how to use `cleanlab` on the simple IRIS dataset.
-* `model_selection_demo.ipynb` - tutorial showing model selection on the cleanlab's parameter settings.
-* `simplifying_confident_learning_tutorial.ipynb` - tutorial implementing cleanlab as raw numpy code.
-* `visualizing_confident_learning.ipynb` - tutorial to demonstrate the noise matrix estimation performed by cleanlab.
+Recommended order of examples to try:
+
+| Notebook                                                                                                                                        | Description                                                                                                            |
+| ----------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| [iris_simple_example.ipynb](https://github.com/cleanlab/examples/blob/master/iris_simple_example.ipynb)                                         | Use `cleanlab` to find synthetic label errors in the Iris dataset.                                                     |
+| [classifier_comparison.ipynb](https://github.com/cleanlab/examples/blob/master/classifier_comparison.ipynb)                                     | Demonstrate how `cleanlab` can be used to train 10 different classifiers on 4 dataset distributions with label errors. |
+| [model_selection_demo.ipynb](https://github.com/cleanlab/examples/blob/master/model_selection_demo.ipynb)                                       | Perform hyperparameter optimization with `cleanlab`'s hyperparameters.                                                 |
```

Comment:
[Nit] hyperparameter optimization to find the best settings of  `cleanlab`'s optional parameters.

Reply:
That sounds better. I changed it to this description. Thanks.

9f1a549f924f08c474b0b8bdd1d9c62c5fa3f0f4